### PR TITLE
docs(NODE-4158): Annotate BSON classes with a category

### PR DESCRIPTION
**Pull request by kampofo**
https://jira.mongodb.org/browse/NODE-4158

### Double check the following

- [ ] Ran `npm run lint` script
- [ ] Self-review completed using the [steps outlined here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/CONTRIBUTING.md#reviewer-guidelines)
- [ ] PR title follows the correct format: `<type>(NODE-xxxx)<!>: <description>`
- [ ] Changes are covered by tests
- [ ] New TODOs have a related JIRA ticket

### CODE REVIEW

**Review by baileympearson — APPROVED** [review]
First PR on the Node team!  🎉 🥳

**Comment by baileympearson** [comment]
Screen Shot 2022-04-20 at 10 16 19 AM


BSON types now show up grouped by class

File: src/map.ts [hunk 1]
```diff
@@ -3,7 +3,9 @@
 
 import { getGlobal } from './utils/global';
 
-/** @public */
+/**
+ * @public
+ */
```
Comment (nbbeeken) [inline src/map.ts:8]:
```suggestion
/** @public */
```

**Review by nbbeeken — CHANGES_REQUESTED** [review]
small nit :) nice work!

**Review by nbbeeken — APPROVED** [review]

**Pull request merged**